recipes/cluster_master.rb

```diff
@@ -13,11 +13,10 @@
 ## Recipes
 include_recipe 'cerner_splunk::_install_server'
 
-password_file = File.join node['splunk']['external_config_directory'], 'password'
-
-execute 'apply-cluster-bundle' do
-  command "cat #{password_file} | xargs #{node['splunk']['cmd']} apply cluster-bundle --answer-yes -auth admin:"
+execute 'apply-cluster-bundle' do # ~FC009
+  command lazy { "#{node['splunk']['cmd']} apply cluster-bundle --answer-yes -auth admin:#{node.run_state['cerner_splunk']['admin-password']}" }
   environment 'HOME' => node['splunk']['home']
+  sensitive true
```

Comment:
The command itself now contains the admin password for the node. Thus sensitive. 

A future change may be to manage the Splunk session, so we don't have to pass the admin password with every command.